CMake/HPHPFindLibs.cmake

```diff
@@ -29,6 +29,9 @@ endif()
 
 # boost checks
 find_package(Boost 1.51.0 COMPONENTS system program_options filesystem context REQUIRED)
```

Comment:
The `REQUIRED` here means that CMake should be failing immediately if it isn't found.